### PR TITLE
updated the article with a new GIF and style guide changes

### PR DESCRIPTION
**Pull request by jonat123**

### CODE REVIEW

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 1]
```diff
@@ -4,7 +4,10 @@ versionFrom: 10.0.0
 
 # Migrating an Existing Site to Umbraco Cloud
 
-Sometimes you may already have a Umbraco site built that did not start with a clone of a Umbraco Cloud site. Or perhaps you have decided to move a site that's already live on Umbraco Cloud. In any case, migrating an existing site is not difficult, but it does require some specific steps, and an understanding of how Umbraco Cloud deployments work can be very helpful.
+Sometimes you may already have a Umbraco site built that did not start with a clone of a Umbraco Cloud site.
+Or perhaps you have decided to move a site that's already live on Umbraco Cloud.
+
+Migrating an existing site is not difficult, however it require some specific steps, and an understanding of how Umbraco Cloud deployments work, can be helpful.
```
Comment (sofietoft) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:10]:
```suggestion
Migrating an existing site requires some specific steps and an understanding of how Umbraco Cloud deployments work can be helpful.
```

**Review by sofietoft — COMMENTED** [review]
Thanks for the PR @jonat123 !

It's like the Gif cuts off when you enter the phone number?
Or is that intentional? 🤔

**Comment by jonat123** [comment]
@sofietoft That is not intentional at all 😮 

Not sure what happened... Will see if i can fix it!

**Comment by jonat123** [comment]
@sofietoft The GIF has been fixed :)

**Comment by sofietoft** [comment]
Looks great @jonat123 💪 
Let's get this merged in then!

**Pull request merged**